### DISCUSSION TITLE
Can we get a synapse like launcher?

### DISCUSSION THREAD

**Discussion by ritulahkar** [linuxmint/discussions#342] [Ideas]
The age old launcher synapse is great but no longer maintained. Can we have a similar applet?

**Comment by rcalixte** [comment]
There is [Cinnamenu](https://cinnamon-spices.linuxmint.com/applets/view/322) depending on the features you want. There is also [Ulauncher](https://ulauncher.io/) if you want an application that uses a similar backend engine to Synapse but is still maintained.

**Reply by ritulahkar** [reply]
I tried Ulauncher but it's functionality is less compared to synapse. Like synapse knows last active files. We can easily launch resent files from synapse. Play recent music. And it has nice categorical search. Synapse can work as one stop for most common tasks. 

That is why we are hopeful that having a native applet with synapse like functionality will be hopeful.

**Reply by Secret-chest** [reply]
Why not bind a keyboard shortcut to launch one of them?

**Comment by Jopp-gh** [comment]
I agree with the OP. 

Launchers are very convenient, Mint's launcher is useful and handy, but some upgrade wouldn't hurt, with features like:

- workspaces (custom list(s) of apps launching together, 
eg for office, surfing, entertainment,  etc.  ) like the app *Workspaces* 
- a calculator !

**Design**
a nice search mask (font-size 24px or more) at the center of the screen, instead of a tall, ms-windows like panel living in the menu bar 

Transforming Mint's launcher in Ulauncher/Albert/Synapse is not feasible, imo. Unless developers don't agree and start a collaboration that benefits only Mint.

**Reply by ritulahkar** [reply]
I do use and like Cinnamenu. I want it to stay. But I use synapse for quick things. I like it's concept and functionality. It keep track of recent files, recent apps. It's very helpful. Category wise search. I am yet to find a launcher that is on par with synapses. But it lacks option for in place met search. That is little disappointing.

**Comment by moloch1994** [comment]
Not exactly a solution, but the Korean distro HamoniKR (based on Mint and a sponsor of it) comes with Albert launcher pre-installed, so this can be an option for those users who want this type of launcher out of the box.
